### PR TITLE
merge release instructions

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
Turns out splitting the file (#13160) was not such a good idea.

Also tweak the text a little bit, hopefully making it clearer.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

**Review by sofiafaro-da — APPROVED** [review]
Thanks!

**Comment by stefanobaghino-da** [comment]
> Turns out splitting the file (#13160) was not such a good idea.

Can you expand on the reason why it was not such a good idea?

File: release/RELEASE.md [hunk 1]
```diff
@@ -14,6 +13,108 @@ patches we backport to the 1.0 release branch).
...
+
+## Making a Snapshot Release (2.x)
+
+1. Open a PR on the [DACH-NY/assembly] repo using the most recent Canton
+   snapshot. See the instructions in the README of the [DACH-NY/assembly] repo
+   for details.
+
+1. Merge the PR and wait for the corresponding `main` build to finish.
+
+1. Go to the [Testing](testing) section of this file.
```
Comment (S11001001) [inline release/RELEASE.md:27]: Link broken on GH at least. Maybe...?

```markdown
[Testing](#testing)
```

File: release/RELEASE.md [hunk 2]
```diff
@@ -14,6 +13,108 @@ patches we backport to the 1.0 release branch).
...
+   @ray-roestenburg-da or @adriaanm-da.
+
+1. Merge the PR and wait for the corresponding `main` build to finish.
+
+1. Go to the [Testing](testing) section of this file.
+
+## Making a Snapshot Release (1.x)
+
+At this point, snapshot releases on 1.x should be rare, and only meant to be
+precursors to stable, patch releases. First, identify wich release branch
```
Comment (S11001001) [inline release/RELEASE.md:63]: s/wich/which/

**Comment by garyverhaegen-da** [comment]
> > Turns out splitting the file (#13160) was not such a good idea.
> 
> Can you expand on the reason why it was not such a good idea?

@stewSquared, the only person to actually test the new split instructions with fresh eyes, simply didn't find them. Having everything in one file seems a lot better from that perspective.

Also: 🤦, link should have been https://github.com/digital-asset/daml/pull/12820.

**Comment by sofiafaro-da** [comment]
@garyverhaegen-da For the record, I also tested the split release process with fresh eyes and found it confusing.

**Comment by stefanobaghino-da** [comment]
The hope was that each file would have been easy to follow linearly instead of having a series of conditions at various steps, but I guess that didn't pan out as I hoped.

**Pull request merged**